test/bench/codec.bench.ts

```diff
@@ -0,0 +1,46 @@
+import { itBench } from '@dapplion/benchmark'
+import { Flag, FrameHeader, FrameType } from '../../src/frame.js'
+import { encodeFrame } from '../../src/encode.js'
+import { decodeHeader } from '../../src/decode.js'
+import { decodeHeaderNaive, encodeFrameNaive } from '../codec.util.js'
+
+describe('codec benchmark', () => {
+  for (const { encode, name } of [
+    { encode: encodeFrame, name: 'encodeFrame' },
+    { encode: encodeFrameNaive, name: 'encodeFrameNaive' }
+  ]) {
+    itBench<FrameHeader, undefined>({
+      id: `encode a frame header - ${name}`,
+      beforeEach: () => {
+        return {
+          type: FrameType.WindowUpdate,
+          flag: Flag.ACK,
+          streamID: Math.round(Math.random() * 2 ** 32),
+          length: Math.round(Math.random() * 2 ** 32)
```

Comment:
If the number here doesn't matter I think it's best to pick a specific number to reduce entropy in the benchmark